ckanext/qgov/common/resource_type_validation.py

```diff
@@ -0,0 +1,177 @@
+# encoding: utf-8
```

Comment:
Awesome use of compartmentalization. This makes its very easy to split out onto its own plugin by others or ourselves 👍  A+ job @ThrawnCA